onnx/cpp2py_export.cc

```diff
@@ -109,71 +110,80 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
     return *schema;
   });
 
-  defs.def(
-      "get_all_schemas",
-      []() -> const std::vector<OpSchema> {
-        return OpSchemaRegistry::get_all_schemas();
-      });
+  defs.def("get_all_schemas", []() -> const std::vector<OpSchema> {
+    return OpSchemaRegistry::get_all_schemas();
+  });
 
-  defs.def(
-      "get_all_schemas_with_history",
-      []() -> const std::vector<OpSchema> {
-        return OpSchemaRegistry::get_all_schemas_with_history();
-      });
+  defs.def("get_all_schemas_with_history", []() -> const std::vector<OpSchema> {
+    return OpSchemaRegistry::get_all_schemas_with_history();
+  });
 
   // Submodule `checker`
   auto checker = onnx_cpp2py_export.def_submodule("checker");
   checker.doc() = "Checker submodule";
 
-  py::class_<checker::CheckerContext> checker_context(checker, "CheckerContext");
-  checker_context
-      .def(py::init<>())
-      .def_property("ir_version", &checker::CheckerContext::get_ir_version, &checker::CheckerContext::set_ir_version)
-      .def_property("opset_imports", &checker::CheckerContext::get_opset_imports, &checker::CheckerContext::set_opset_imports);
+  py::class_<checker::CheckerContext> checker_context(
+      checker, "CheckerContext");
+  checker_context.def(py::init<>())
+      .def_property(
+          "ir_version",
+          &checker::CheckerContext::get_ir_version,
+          &checker::CheckerContext::set_ir_version)
+      .def_property(
+          "opset_imports",
+          &checker::CheckerContext::get_opset_imports,
+          &checker::CheckerContext::set_opset_imports);
 
   py::register_exception<checker::ValidationError>(checker, "ValidationError");
 
   checker.def(
-      "check_value_info", [](const py::bytes& bytes, const checker::CheckerContext& ctx) -> void {
+      "check_value_info",
+      [](const py::bytes& bytes, const checker::CheckerContext& ctx) -> void {
         std::unique_ptr<ValueInfoProto> proto(new ValueInfoProto());
         ParseProtoFromPyBytes(proto.get(), bytes);
         checker::check_value_info(*proto, ctx);
       });
 
   checker.def(
-      "check_tensor", [](const py::bytes& bytes, const checker::CheckerContext& ctx) -> void {
+      "check_tensor",
+      [](const py::bytes& bytes, const checker::CheckerContext& ctx) -> void {
         std::unique_ptr<TensorProto> proto(new TensorProto());
         ParseProtoFromPyBytes(proto.get(), bytes);
         checker::check_tensor(*proto, ctx);
       });
 
   checker.def(
-      "check_attribute", [](const py::bytes& bytes, const checker::CheckerContext& ctx) -> void {
+      "check_attribute",
+      [](const py::bytes& bytes, const checker::CheckerContext& ctx) -> void {
         std::unique_ptr<AttributeProto> proto(new AttributeProto());
         ParseProtoFromPyBytes(proto.get(), bytes);
-        checker::check_attribute(*proto, ctx);
+        checker::LexicalScopeContext lex_ctx;
```

Comment:
nit: you can make it const or just create an anonymous one in the function call next line.